### PR TITLE
submission

### PR DESCRIPTION
**Pull request by jameswilsterman**
*Description of changes:*

I am submitting a new task definition for the skill Bedtime Stories by Volley Inc.

The task is named "VolleyBedtimeStories". The task is designed to be requested from other Alexa skills which would like to allow their customers to hear a Volley Bedtime Story.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: BedtimeStories/tasks/VolleyBedtimeStories.1.json
Comment (rongzhoushen) [inline BedtimeStories/tasks/VolleyBedtimeStories.1.json:53]: Can you describe why you would need the requesterSkillId? We don't normally recommend having a requesterSkillId being passed around. If this is about playing a bedtime story, would the name of the bedtime story be more relevant or the preferences of a bedtime story rather than the skill that's requesting?

File: BedtimeStories/tasks/VolleyBedtimeStories.1.json
Comment (rongzhoushen) [inline BedtimeStories/tasks/VolleyBedtimeStories.1.json:62]: When this is false but returned 200, what does that mean?

**Comment by rongzhoushen** [comment]
Thank you for your custom task submission, I have given some comments and questions, can you take a look?

File: BedtimeStories/tasks/VolleyBedtimeStories.1.json
Comment (jameswilsterman) [reply BedtimeStories/tasks/VolleyBedtimeStories.1.json:53]: We'd like to personalize the intro messages and story selections depending on which skill you are coming in from. i.e. if you are coming from our Love Taps skill we will highlight more romance-focused Bedtime Stories vs. if you are coming from Song Quiz or Yes Sire. Essentially, this is a way for us customize the experience. I have a related question, though, will this task be public by default? i.e. will other vendors be able to start using the task as soon as it is approved? If so, would be good data for us to know what other skills are trying to request a bedtime story.

Comment (rongzhoushen) [reply BedtimeStories/tasks/VolleyBedtimeStories.1.json:53]: I see, thanks for the clarification. Once we approve and ingest your schema, it will only be accessible by the vendors in our private beta. In the near future, you have the option of making the task vendor-private, which only the skills within your vendor can access and see.

File: BedtimeStories/tasks/VolleyBedtimeStories.1.json
Comment (rongzhoushen) [reply BedtimeStories/tasks/VolleyBedtimeStories.1.json:62]: Actually, this is not a blocker.

File: BedtimeStories/tasks/VolleyBedtimeStories.1.json
Comment (jameswilsterman) [reply BedtimeStories/tasks/VolleyBedtimeStories.1.json:53]: great! thank you

Comment (rongzhoushen) [reply BedtimeStories/tasks/VolleyBedtimeStories.1.json:53]: Let me know if the above answer helps. Feel free to resolve the conversation, then I'll merge in the changes and ingest the schema. Please note that ingesting the task schema may take 2-3 business days.

Comment (rongzhoushen) [reply BedtimeStories/tasks/VolleyBedtimeStories.1.json:53]: By the way, please also share with us your skill ID.

**Review by rongzhoushen — APPROVED** [review]

**Pull request merged**
